packages/editor/src/components/block-settings-menu/content-only-settings-menu.native.js

```diff
@@ -0,0 +1,3 @@
+export default function ContentOnlySettingsMenu() {
```

Comment:
Don't show in the RN environment. I'm not sure if it's needed though.

Reply:
The file this component is used in for the web (packages/editor/src/components/provider/index.js) has a native version of its own that doesn't use the component, so I don't think this is needed.

Reply:
Turned out that the native provider is using the universal provider so we still need this.